### PR TITLE
GOOGLEDOCS-482: Updated googledocs reference in pom.xml to 3.2.0-A4

### PR DESCRIPTION
**Pull request by aymanthefirst**

### CODE REVIEW

**Comment by montgolfiere** [comment]
Note: targeting ACS 6.2.1 (in dev)

File: packaging/docker/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <properties>
         <dependency.alfresco-mmt.version>6.0</dependency.alfresco-mmt.version>
         <!-- Alfresco GoogleDocs integration version -->
-        <alfresco.googledocs.version>3.1.0</alfresco.googledocs.version>
+        <alfresco.googledocs.version>3.2.0-A4</alfresco.googledocs.version>
```
Comment (PunchedChimera) [inline packaging/docker/pom.xml]: In 3.2 the artifact names have been changed. The new artifact is called `alfresco-googledrive-share`. See line 37

Comment (montgolfiere) [reply packaging/docker/pom.xml]: Yes, as per https://github.com/Alfresco/googledrive/blame/master/_ci/prepare_staging_deploy.sh#L28-L29(GOOGLEDOCS-464).

**Comment by montgolfiere** [comment]
Green build ... I suggest one more approve & we "Squash and Merge" ?

**Pull request merged**